x-pack/plugins/observability_solution/slo/public/pages/slos/components/badges/slo_indicator_type_badge.tsx

```diff
@@ -50,13 +50,10 @@ export function SloIndicatorTypeBadge({ slo, color }: Props) {
               kqlQuery: `slo.indicator.type: ${slo.indicator.type}`,
             });
           }}
-          onClickAriaLabel={i18n.translate(
-            'xpack.observability.slo.indicatorTypeBadge.clickToFilter',
```

Comment:
I noticed that this copy `xpack.observability.slo.indicatorTypeBadge.clickToFilter` is not in the [translation file](https://raw.githubusercontent.com/elastic/kibana/main/x-pack/plugins/translations/translations/ja-JP.json). Shall I contact somebody from the translation team? @maryam-saeidi I have seen you had some conversations in the past regarding translations. Any point of contact I could use?

Reply:
When was `xpack.observability.slo.indicatorTypeBadge.clickToFilter` added? Translation happens after the feature freeze if I remember correctly, so if it is added in v8.14, it is OK that we don't have a translation.

Reply:
Ok good to know. Let me check!

Reply:
Looks like it was introduced [here](https://github.com/elastic/kibana/pull/176339/files#diff-c1b5d6134700ceed1ddaf8d405ced72400afef9270ff5739dd4fe56a7892f366R54) and I see PR was targeted for `8.13`

Reply:
I am thinking that maybe because file is too big, content is not being highlighted. I will check out main and check the file locally in my machine and verify if above copy exists in the translation file

Reply:
I clarified with the translation team and they didn't merge the 8.13 translation files into kibana yet. So all good